pkg/apis/navigator/types.go

```diff
@@ -32,6 +32,7 @@ type CassandraClusterSpec struct {
 	NodePools []CassandraClusterNodePool
 	Version   version.Version
 	Image     *ImageSpec
+	Paused    *bool
```

Comment:
Does this need to be a pointer? (i.e. is there a 3rd 'unset' state that we need to support?).

IMO `paused: false` and `paused: true` are expressive enough?

Reply:
I did this so that omitempty hides the field when it's not set, but I don't think that's actually necessary. I'll change it to `bool`